### PR TITLE
[EMP-850] r1 new restore image

### PR DESCRIPTION
**Pull request by raysiudak**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -432,6 +441,14 @@ while getopts ":ch" options; do
             rm -rf "${BUILD_OUTPUT_DIR}"
         fi
         echo "Cleaned up '${BUILD_OUTPUT_DIR}'."
+        if [ -d "${BB_DIR}" ] && [ -z "${BB_DIR##*busybox-*}" ]; then
+            rm -rf "${BB_DIR}"
+        fi
```
Comment (alexborro) [inline build.sh]: Is it needed to check if the DIR exists if you are gonna remove it?? I would just remove the if.

File: build.sh [hunk 2]
```diff
@@ -41,15 +41,20 @@ BUILDCONFIG="opinicus"
 # Setup internal variables
 KCONFIG="${CWD}/configs/${BUILDCONFIG}_config"
 KERNEL_BUILD_DIR="${CWD}/_build_armhf/${BUILDCONFIG}-linux"
+KERNEL_IMAGE="uImage-sun7i-a20-opinicus_v1"
 BUILD_OUTPUT_DIR="${CWD}/_build_armhf/"
 DEBIAN_DIR="${BUILD_OUTPUT_DIR}/debian"
 BOOT_FILE_OUTPUT_DIR="${DEBIAN_DIR}/boot"
 
 INITRAMFS_MODULES_REQUIRED="sunxi_wdt.ko ssd1307fb.ko drm.ko sun4i-backend.ko sun4i-drm.ko sun4i-tcon.ko sun4i-drm-hdmi.ko sun4i-hdmi-i2c.ko"
+
```
Comment (alexborro) [inline build.sh]: All this modules are needed?? Including ssd1307fb???

File: build.sh (hunk 1 above)
Comment (raysiudak) [reply build.sh]: Well, that is where all shit could hit the fan.. what if BB_DIR contained "/" ??

Comment (alexborro) [reply build.sh]: Agreed, but for that you need to edit the script and put a " / " into BB_DIR. You will learn before committing to pay attention when typing ;-) Anyway, I just wonder if is it better to complicate things just to add some safe measures that is improbable to happen.

File: build.sh (hunk 2 above)
Comment (raysiudak) [reply build.sh]: Since support is dropped for UM3, the ssd1307fb can be dropped.

**Pull request merged**